specification/iothub/resource-manager/Microsoft.Devices/preview/2018-12-01-preview/iothub.json

```diff
@@ -2386,8 +2386,13 @@
           "minimum": 10485760
         },
         "encoding": {
-          "description": "Encoding that is used to serialize messages to blobs. Supported values are 'avro' and 'avrodeflate'. Default value is 'avro'.",
-          "type": "string"
+          "description": "Encoding that is used to serialize messages to blobs. Supported values are 'avro', 'avrodeflate', and 'JSON'. Default value is 'avro'.",
```

Comment:
I guess the values described in the "Supported values are" should probably match the enum casing

Reply:
There are multiple instances where the values described in the "Supported values are" do no match the enum casing in the repo. Is this a requirement?

In addition, do you understand the failure for "Automatin for azure-sdk-for-go". - "FATAL: System.InvalidOperationException: codegen for preview swagger Microsoft.Devices/preview/2018-12-01-preview/iothub.json must be under a preview subdirectory"

My change is in a preview subdirectory. Would you know what I am missing? Should I create a subdirectory for this month and year?

Reply:
Sorry, I'm not an MS employee, I've just been trying to get some automated spellchecking landed :wink: 
I've seen some failures like that on my own PRs. Sometimes it's because the folder isn't created yet in the upstream repo (or something like that).
I'm sure one of the real admins can point you in the right direction though